src/astar/common.js

```diff
@@ -31,20 +31,22 @@ export const scheduleTask = async ({
     const astarAdapter = new AstarAdapter(astarApi, astarConfig);
     await astarAdapter.initialize();
 
-    const oakChainData = oakAdapter.getChainData();
-    const astarChainData = astarAdapter.getChainData();
+    const oakChainData = oakAdapter.getChainConfig();
+    const astarChainData = astarAdapter.getChainConfig();
+
+    const [astarDefaultAsset] = astarChainData.assets;
```

Comment:
Looking at this line, we probably should include the defaultAsset attribute in the chain class, but this PR will work regardless.

Reply:
Yes, we can add defaultAsset to chain.